Uncaught Exception: error: terminating connection due to administrator command
    at Parser.parseErrorMessage (/var/task/api/node_modules/pg-protocol/dist/parser.js:285:98)
    at Parser.handlePacket (/var/task/api/node_modules/pg-protocol/dist/parser.js:122:29)
    at Parser.parse (/var/task/api/node_modules/pg-protocol/dist/parser.js:35:38)
    at TLSSocket.<anonymous> (/var/task/api/node_modules/pg-protocol/dist/index.js:11:42)
    at TLSSocket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TLSWrap.onStreamRead (node:internal/stream_base_commons:189:23)
    at TLSWrap.callbackTrampoline (node:internal/async_hooks:130:17) {
  length: 116,
  severity: 'FATAL',
  code: '57P01',
  detail: undefined,
  hint: undefined,
  position: undefined,
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'postgres.c',
  line: '3289',
  routine: 'ProcessInterrupts',
  client: Client {
    _events: [Object: null prototype] { error: [Function (anonymous)] },
    _eventsCount: 1,
    _maxListeners: undefined,
    connectionParameters: ConnectionParameters {
      user: 'neondb_owner',
      database: 'neondb',
      port: 5432,
      host: 'ep-raspy-cake-ae8qbqnl.c-2.us-east-2.aws.neon.tech',
      binary: false,
      options: undefined,
      ssl: {},
      client_encoding: '',
      replication: undefined,
      isDomainSocket: false,
      application_name: undefined,
      fallback_application_name: undefined,
      statement_timeout: false,
      lock_timeout: false,
      idle_in_transaction_session_timeout: false,
      query_timeout: false,
      connect_timeout: 0
    },
    user: 'neondb_owner',
    database: 'neondb',
    port: 5432,
    host: 'ep-raspy-cake-ae8qbqnl.c-2.us-east-2.aws.neon.tech',
    replication: undefined,
    _Promise: [Function: Promise],
    _types: TypeOverrides { _types: [Object], text: {}, binary: {} },
    _ending: true,
    _ended: false,
    _connecting: false,
    _connected: true,
    _connectionError: false,
    _queryable: false,
    enableChannelBinding: false,
    connection: Connection {
      _events: [Object: null prototype],
      _eventsCount: 23,
      _maxListeners: undefined,
      stream: [TLSSocket],
      _keepAlive: false,
      _keepAliveInitialDelayMillis: 0,
      lastBuffer: false,
      parsedStatements: {},
      ssl: {},
      _ending: true,
      _emitMessage: false,
      _connecting: true,
      [Symbol(shapeMode)]: false,
      [Symbol(kCapture)]: false
    },
    queryQueue: [],
    binary: false,
    processID: -827084387,
    secretKey: -1878418394,
    ssl: {},
    _connectionTimeoutMillis: 0,
    _connectionCallback: null,
    saslSession: null,
    release: [Function (anonymous)],
    activeQuery: null,
    readyForQuery: true,
    hasExecuted: true,
    _poolUseCount: 2,
    [Symbol(shapeMode)]: false,
    [Symbol(kCapture)]: false
  }
}
Node.js process exited with exit status: 129. The logs above can help with debugging the issue.